src/Tickets/Commerce/Hooks.php

```diff
@@ -185,14 +236,55 @@ public function pre_filter_admin_order_table( $query ) {
 			}
 		}
 
+		$date_from = sanitize_text_field( tribe_get_request_var( 'tec_tc_date_range_from', '' ) );
+		$date_to   = sanitize_text_field( tribe_get_request_var( 'tec_tc_date_range_to', '' ) );
```

Comment:
`tribe_get_request_var` should already sanitize the values.

Reply:
It does a sanitization of removing html special chars and removing tags. `sanitize_text_field` covers more stuff so id like to leave it there.